modules/loader-utils/src/lib/worker-utils/worker-pool.js

```diff
@@ -73,11 +73,16 @@ export default class WorkerPool {
   }
 
   _onWorkerDone(worker) {
-    if (this.isDestroyed || !this.reuseWorkers) {
+    if (this.isDestroyed) {
       worker.destroy();
-      this.count--;
```

Comment:
Maybe just `return` here.  And remove the else statement.

```js
if (this.isDestroyed) {
      worker.destroy();
      return;
}
```

Reply:
Fixed